lib/mongo/client_encryption.rb

```diff
@@ -71,14 +71,21 @@ def initialize(key_vault_client, options={})
     #     will not be properly parsed.
     # @option options [ Array<String> ] :key_alt_names An optional array of
     #   strings specifying alternate names for the new data key.
+    # @option options [ BSON::Binary | nil ] :key_material Optional
+    #   96 bytes to use as custom key material for the data key being created.
+    #   If :key_material option is given, the custom key material is used
+    #   for encrypting and decrypting data.
     #
     # @return [ BSON::Binary ] The 16-byte UUID of the new data key as a
     #   BSON::Binary object with type :uuid.
     def create_data_key(kms_provider, options={})
       key_document = Crypt::KMS::MasterKeyDocument.new(kms_provider, options)
+
       key_alt_names = options[:key_alt_names]
-      @encrypter.create_and_insert_data_key(key_document, key_alt_names)
+      key_material = options[:key_material]
+      @encrypter.create_and_insert_data_key(key_document, key_alt_names, key_material)
     end
+    alias create_key create_data_key
```

Comment:
I think style we've been using for aliases is to use symbols?

Something like:
```
alias :create_key :create_data_key
```
Not sure if that makes sense in this context

Reply:
Good point, fixed